### PR TITLE
[prometheus-kafka-exporter] new chart donation

### PR DESCRIPTION
**Pull request by gkarthiks**
#### What this PR does / why we need it:
A new chart donation to prometheus-community

#### Checklist
- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by gkarthiks** [comment]
@scottrigby here's the PR for the kafka-exporter. 

And you are right, since the codeowners file is included in the PR, it seems the chart-admins are automatically added for the reviews.

**Comment by gkarthiks** [comment]
Hi @scottrigby and @torstenwalter can you look into this PR and merge into main?

File: charts/prometheus-kafka-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+```console
+# Helm 3 or 2
+$ helm upgrade [RELEASE_NAME] [CHART] --install
+```
+
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+## Configuration
+
+See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](./values.yaml), or run these configuration commands:
```
Comment (scottrigby) [inline charts/prometheus-kafka-exporter/README.md:61]: See https://github.com/artifacthub/hub/issues/629
```suggestion
See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-kafka-exporter/values.yaml), or run these configuration commands:
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -24,3 +24,4 @@
 /charts/prometheus-redis-exporter/ @acondrat @zanhsieh
 /charts/prometheus-snmp-exporter/ @miouge1
 /charts/prometheus-to-sd/ @acondrat
+/charts/prometheus-kafka-exporter/ @gkarthiks
```
Comment (scottrigby) [inline .github/CODEOWNERS:27]: Can you move this up to be in alphabetical order? Or just run https://github.com/prometheus-community/helm-charts/blob/main/scripts/check-codeowners.sh 😄

File: charts/prometheus-kafka-exporter/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+appVersion: "v1.2.0"
+description: A Helm chart to export the metrics from Kafka in Prometheus format.
+name: prometheus-kafka-exporter
+home: https://gkarthiks.github.io/helm-charts/charts/prometheus-kafka-exporter
+version: 0.1.4
```
Comment (scottrigby) [inline charts/prometheus-kafka-exporter/Chart.yaml:6]: Why does this chart start with `version: 0.1.4`? Is it donated from somewhere else (if so, where?). If not, we should start with `0.1.0`.

**Review by scottrigby — CHANGES_REQUESTED** [review]
nice 😄 a few comments

File: charts/prometheus-kafka-exporter/Chart.yaml (hunk 3 above)
Comment (gkarthiks) [reply charts/prometheus-kafka-exporter/Chart.yaml:6]: yes, its from https://artifacthub.io/packages/helm/gkarthiks/prometheus-kafka-exporter

**Comment by gkarthiks** [comment]
@scottrigby all the comments are addressed. Lemme know if this looks good.

File: charts/prometheus-kafka-exporter/Chart.yaml (hunk 3 above)
Comment (scottrigby) [reply charts/prometheus-kafka-exporter/Chart.yaml:6]: Awesome 👍

Do you mind noting in the description this is from https://github.com/danielqsj/kafka_exporter?

Will you also link a PR there to deprecate that chart (similar to https://github.com/helm/charts/blob/master/PROCESSES.md)?

**Review by scottrigby — APPROVED** [review]
Lgtm

**Comment by scottrigby** [comment]
Do we want to get one more maintainers eyes on this?￼￼ We didn't note an exception in PROCESSES for new charts, but I am not in a position to actually install and test this right now. It just looks good from a code review point of view.￼

**Comment by gkarthiks** [comment]
I have @golgoth31 as the collaborator in my repo. 

Hi @golgoth31 - Would you like to be an additional maintainer for kafka-exporter in Prometheus Community?

**Comment by golgoth31** [comment]
Hi all,
Yes, with pleasure.
Thank you

**Comment by gkarthiks** [comment]
thanks @golgoth31, what would be your email address?

**Comment by golgoth31** [comment]
You can join me at golgoth31@notrenet.com

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -15,6 +15,7 @@
 /charts/prometheus-consul-exporter/ @gkarthiks @timm088
 /charts/prometheus-couchdb-exporter/ @gkarthiks
 /charts/prometheus-druid-exporter/ @iamabhishek-dubey
+/charts/prometheus-kafka-exporter/ @gkarthiks
```
Comment (scottrigby) [inline .github/CODEOWNERS:18]: @gkarthiks please add @golgoth31

```suggestion
/charts/prometheus-kafka-exporter/ @gkarthiks @golgoth31
```

File: charts/prometheus-kafka-exporter/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+home: https://github.com/danielqsj/kafka_exporter
+version: 0.1.4
+sources:
+  - https://gkarthiks.github.io/helm-charts/charts/prometheus-kafka-exporter
+keywords:
+  - prometheus
+  - kafka
+maintainers:
+  - name: gkarthiks
+    email: github.gkarthiks@gmail.com
```
Comment (scottrigby) [inline charts/prometheus-kafka-exporter/Chart.yaml:14]:
```suggestion
  - name: gkarthiks
    email: github.gkarthiks@gmail.com
  - name: golgoth31
    email: golgoth31@notrenet.com
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
@gkarthiks a few changes now

**Comment by scottrigby** [comment]
@prometheus-community/helm-charts-admins Please invite @golgoth31 to `prometheus-community/helm-charts-maintainers`

**Comment by gkarthiks** [comment]
@scottrigby  I am gonna create a new PR because of the DCO chaos :wink:

**Pull request closed without merging**